servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/ManagementClientConstants.java

```diff
@@ -21,13 +21,13 @@
     static String API_VERSION = "2017-04";
     static String API_VERSION_QUERY = "api-version=" + API_VERSION;
 
-    static String ServiceBusSupplementartyAuthorizationHeaderName = "ServiceBusSupplementaryAuthorization";
-    static String ServiceBusDlqSupplementaryAuthorizationHeaderName = "ServiceBusDlqSupplementaryAuthorization";
-    static String HttpErrorSubCodeFormatString = "SubCode=%s";
-    static String ConflictOperationInProgressSubCode =
-        String.format(HttpErrorSubCodeFormatString, ExceptionErrorCodes.ConflictOperationInProgress);
-    static String ForbiddenInvalidOperationSubCode =
-        String.format(HttpErrorSubCodeFormatString, ExceptionErrorCodes.ForbiddenInvalidOperation);
+    static String serviceBusSupplementartyAuthorizationHeaderName = "ServiceBusSupplementaryAuthorization";
+    static String serviceBusDlqSupplementaryAuthorizationHeaderName = "ServiceBusDlqSupplementaryAuthorization";
+    static String httpErrorSubCodeFormatString = "SubCode=%s";
```

Comment:
Make them all final as they are constants. Then you don't have to rename them to camelCase.